05/app.js

```diff
@@ -1,3 +1,40 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+const para = document.createElement('p')
+const firstArticle = document.createElement('article')
+firstArticle.closest('section')
+
+console.log(firstArticle);
+
+const btn = document.createElement('button')
+curr.parentElement.append(btn)
+
+const siblingOne = curr.previousElementSibling
+const siblingTwo = siblingOne.previousElementSibling
+
+const article = curr.parentElement.nextElementSibling
+
+
+
+article.setAttribute('title', 'nextElementSibling')
+const copyArticle = article.children
```

Comment:
Kopia elementu to on sam + zawartość więc same "dzieci" to za mało.

Tutaj należy użyć `article.cloneNode(true)`; 
Jeśli natomiast chcesz wykorzystać to do określenia rodzeństwa to dobry kierunek.
Teraz nalezałoby zrobić pętle, tylko w takim wypadku należało by nazwać zmianną "children" zamiast "copyArticle".